### DISCUSSION TITLE
MsgPack support

### DISCUSSION THREAD

**Discussion by lucaspataro** [miguelgrinberg/Flask-SocketIO#1806] [General]
Hello, 

since [python-socketio](https://github.com/miguelgrinberg/python-socketio) supports MessagePack since a few versions i am just curios if there is a timeline when this will be integrated into Flask-SocketIO,

Thanks for your great work on these libraries :).

**Comment by miguelgrinberg** [comment]
Have you tried adding `serializer='msgpack'` when you initialize the extension? All arguments that aren't recognized are passed down to python-socketio, so there is really nothing that needs to be done in this package, I think.

**Reply by lucaspataro** [reply]
Thanks for the response. 

Ah okay, i wasn't aware of that. I didn't try it yet, bc i am still in the resource gathering state before implementing a new feature in our app. I will give it a try when i start implementation. 👍
